Bento/Views/Adapters/AdapterProtocol.swift

```diff
@@ -33,8 +33,4 @@ extension AdapterStoreOwner {
         get { return store.cachesSizeInformation }
         set { store.cachesSizeInformation = newValue }
     }
-
-    public func invalidateSize(at indexPath: IndexPath) {
```

Comment:
Moved this to the actual conformance site because the runtime didn't manage to pick this protocol extension provided implementation up, while the compiler reckons it having satisfied the protocol requirement. 😬